ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1453,13 +1458,21 @@ def test_ipa_healthcheck_renew_internal_cert(self):
         This testcase checks that CADogtagCertsConfigCheck can handle
         cert renewal, when there can be two certs with the same nickname
         """
-        if (tasks.get_pki_version(self.master) < tasks.parse_version('11.4.0')):
+        if (tasks.get_pki_version(
+                self.master) < tasks.parse_version('11.4.0')):
             raise pytest.skip("PKI known issue #2022561")
-        self.master.run_command(['ipa-cacert-manage', 'renew', '--self-signed'])
-        returncode, data = run_healthcheck(
-            self.master,
-            "pki.server.healthcheck.meta.csconfig",
-            "CADogtagCertsConfigCheck",
+        elif (tasks.get_pki_version(
+                self.master) >= tasks.parse_version('11.5.0')):
+            raise pytest.skip("Skipping test for 11.5 pki version, since"
+                              "check CADogtagCertsConfigCheck is"
+                              "not present in source "
+                              "pki.server.healthcheck.meta.csconfig")
+        else:
```

Comment:
The "else:" is not needed because pytest.skip would have already exited from the function. Remove the line with else and realign the remaining code.